.github/workflows/ci.yml

```diff
@@ -488,7 +494,11 @@ jobs:
       - name: install wasm target
         run: rustup target add wasm32-unknown-unknown
       - name: Build WASM for ${{ matrix.wasm-crate }}
```

Comment:
Some of the titles are partially blank, I think this needs"
`${{ matrix.wasm-crate }}` -> `${{matrix.context.crate}}`